build-aux/m4/bitcoin_qt.m4

```diff
@@ -123,7 +123,7 @@ AC_DEFUN([BITCOIN_QT_CONFIGURE],[
     _BITCOIN_QT_CHECK_STATIC_LIBS
 
     if test "x$qt_plugin_path" != x; then
-      QT_LIBS="$QT_LIBS -L$qt_plugin_path/platforms"
+      QT_LIBS="$QT_LIBS -L$qt_plugin_path/platforms -L$qt_plugin_path/styles"
```

Comment:
nit (leave for a follow up): "style" directory is not present on all platforms
```suggestion
      QT_LIBS="$QT_LIBS -L$qt_plugin_path/platforms"
      if test -d "$qt_plugin_path/styles"; then
        QT_LIBS="$QT_LIBS -L$qt_plugin_path/styles"
      fi
```

Reply:
Ok. We can combine this as well as [this](https://github.com/bitcoin/bitcoin/pull/21376#discussion_r590693514), and other cleanups into a followup.